src/components/AppProvider/AppProvider.scss

```diff
@@ -22,7 +22,7 @@ button {
 
 html {
   position: relative;
-  font-size: ($base-font-size / $default-browser-font-size) * 100%;
```

Comment:
Are these vars used elsewhere? Maybe we can 🔥 one or both

Reply:
They are but will be removed soon! Possibly as part of https://github.com/Shopify/polaris-react/issues/4777